### DISCUSSION TITLE
Hard to slide left and right across a file inside of a repository

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#6699] [Mobile]
Whenever I try to slide left or right on a file (i.e. when the file's width is too large) in the Android app, it's quite hard to actually make it do so. Most of the time sliding doesn't work, and instead requires me to slide at just the right angle for it to work.

**Comment by brianlovin** [comment]
Thanks for the note @hwittenborn — I'm trying this on a couple of my test devices, and I'm not quite able to reproduce any issues here. Could you share more details about:

— your device
— where you're viewing the file from (e.g. from a pull request, the file browser, or something else)

Thanks!

**Reply by Condo-Colin** [reply]
Hello!
I think this is the issue I came here to report.
When panning the view of code in a file opened in the app, we need to be able to pan in 2 dimensions simultaneously.
Currently we can only scroll vertically or horizontally, not both together.
Depending on which scroll direction the app detects first on each touch, that is what we're limited to for that duration.
Please advise if you're able to correct this action, or at least make it an option.
Thanks very much!

**Reply by hwittenborn** [reply]
Well for me personally I think it makes sense to only go in one direction as a time, as otherwise it could make it quite easy to make it look all jiggly-like when viewing a code file.

Disregarding that I still need to figure out why the issue's happening. I'm wanting to get a small video together, but I haven't gotten around to it yet.

**Reply by Drumber** [reply]
Hi, I need to bring up this issue again here as it has been annoying me for months now. I also use an LG phone (LG V30, Android 9) and I can't scroll horizontally in the code explorer either (or only when trying really hard, see video).

What I found out:
- Problem seems to occur only on LG phones (same problem [here](https://github.community/t/github-android-app/217937)).
- Horizontal scrolling does not work in the code explorer, but works fine in README files (see video).
- The only apps I know of with this problem are GitHub Mobile and the Weather widget in the Google app (see also [here](https://www.reddit.com/r/lgg7/comments/pnqspb/google_weather_scroll_problem/)).

Even though LG phones are pretty much dead now, I still hope that someone will look at this issue, since it probably still affects some people.

https://user-images.githubusercontent.com/29163322/180164362-b6098972-e481-47aa-aa20-d750a7c8b6f1.mp4

https://user-images.githubusercontent.com/29163322/180231334-1abdc4f8-544f-445d-9710-0d783f375a9f.mp4

**Reply by hwittenborn** [reply]
@Drumber That's pretty much the exact issue I had in that video. I've since moved to a Samsung phone where the issue doesn't happen though, so it definitely seems like something with LG phones.

**Comment by hwittenborn** [comment]
Sorry for the delayed response on my end, I got a bit distracted by other stuff :P

Here's all the requested information though:

Device: LG LML212VL, running Android 8.1.0
File location: Using the file browser from inside a repository's files. Viewing [this file](https://github.com/makedeb/makedeb/blob/alpha/src/makedeb/main.sh) in the app reproduces the issue if you need a concrete example.

**Reply by 01e9** [reply]
Android 12 (LG V60) very frustrating to scroll horizontally. It works rarely, most of tries I get code line highlighted (the line highlighting is too aggressive and cancels the horizontal scrolling)
